arch/Etca/EtcaModule.c

```diff
@@ -0,0 +1,41 @@
+/* Capstone Disassembly Engine */
+/* By Alexander Nutz, 2025 */
+
+#ifdef CAPSTONE_HAS_ETCA
+
+#include "../../cs_priv.h"
+#include "EtcaDisassembler.h"
+#include "EtcaInstPrinter.h"
+#include "EtcaModule.h"
+
+cs_err Etca_global_init(cs_struct *ud)
+{
+	etca_info *info;
+
+	info = cs_mem_malloc(sizeof(etca_info));
```

Comment:
```suggestion
	info = cs_mem_calloc(sizeof(etca_info));
```

Prevented many bugs in the past.

Reply:
oh good catch
while reading trough the code, I assumed that it zero allocates...